event_lead_event/wizard/lead_event.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models, fields
+
+
+class LeadEventConfirm(models.TransientModel):
+
+    _name = "event.event.leadconfirm"
+
+    def _get_current_lead_event(self):
+        return self.env['event.event'].search([('lead_event', '=', True)])
+
+    def _get_event_id(self):
+        event_id = self.env['event.event'].browse(
+            self.env.context['active_id']
+        )
+        if event_id and event_id.lead_event is True:
```

Comment:
keep it as before, set both fields, then check if they are not the same on submit, not here